PnRPython/run_sc_dc_dc_converter.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export DESIGN=sc_dc_dc_converter
+
+./gen_viewer_json.py -b $DESIGN -o ../Viewer/INPUT -d $ALIGN_WORK_DIR/$DESIGN/pnr_output/Results -l INFO  --json_dir $ALIGN_WORK_DIR/$DESIGN/pnr_output/inputs/ --draw_grid
```

Comment:
@sapatnekar, did you intentionally not include the DRC check?

Reply:
I'll fix it up in my next commit. (Don't want to run CI again.)